js/rpc/README.md

```diff
@@ -0,0 +1,103 @@
+# Javascript Custom RPC
+
+<!-- PROJECT SHIELDS -->
+<!--
+*** I'm using markdown "reference style" links for readability.
+*** Reference links are enclosed in brackets [ ] instead of parentheses ( ).
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+
+
+# MRC Custom RPC and Types for Polkadot JS API
+
+An easy way to get all the custom rpc and types config to be able to easily use [MRC](https://github.com/LibertyDSNP/mrc/) with the [Polkadot JS API library](https://www.npmjs.com/package/@polkadot/api).
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+- `npm install @polkadot/api` (Polkadot API Library)
+- `npm install @dsnp/mrc-rpc`
+
+### Usage
+
+For details on use, see the [Polkadot API library documentation](https://polkadot.js.org/docs/api).
+
+```javascript
+// es6 style imports
+import { ApiPromise } from '@polkadot/api';
+import { rpc, types } from "@dsnp/mrc-rpc";
+// ...
+
+const mrcAPI = await ApiPromise.create({
+    // ...
+    rpc,
+    types,
+});
+```
+
+```javascript
+// commonjs require
+const { ApiPromise } = require('@polkadot/api');
+const { rpc, types } = require("@dsnp/mrc-rpc");
+// ...
+
+const mrcAPI = await ApiPromise.create({
+    // ...
+    rpc,
```

Comment:
Just checking: I think this depends on what is the type of rpc being returned but previously we had to wrap it inside an object
```
rpc: {
      ...rpc,
},
```